dhis-2/dhis-services/dhis-service-setting/src/main/java/org/hisp/dhis/setting/SettingKey.java

```diff
@@ -168,6 +168,7 @@
     ANALYTICS_HIDE_WEEKLY_PERIODS( "keyHideWeeklyPeriods", Boolean.FALSE, Boolean.class ),
     ANALYTICS_HIDE_MONTHLY_PERIODS( "keyHideMonthlyPeriods", Boolean.FALSE, Boolean.class ),
     ANALYTICS_HIDE_BIMONTHLY_PERIODS( "keyHideBiMonthlyPeriods", Boolean.FALSE, Boolean.class ),
+    COUNT_DASHBOARD_VIEWS_OF_FAVORITES( "keyCountDashboardViewsOfFavorites", Boolean.FALSE, Boolean.class ),
```

Comment:
The name could be improved on, as i) we don't refer to favorites rather analytical objects in the source code and ii) this relates to the "data statistics" feature and endpoint, which is about more than simply counting.

Another name could be e.g. `ENABLE_ANALYTICAL_OBJECT_STATISTICS_IN_DASBOARDS`.

Reply:
@larshelge Both points well-taken. Just to be clear that we're not enabling the viewing of statistics in dashboards with this setting but gathering them there, I suggest GATHER_ANALYTICAL_OBJECT_STATISTICS_IN_DASHBOARD_VIEWS.